### DISCUSSION TITLE
Request best video parameter for skydiving activity

### DISCUSSION THREAD

**Discussion by Lionel84330** [gopro/labs#1213] [Feature Requests]
Hello, 
I’m starting in video and I practice skydiving. 
In this discipline, some conditions are difficult as the noise of the wind, the brightness very important and a maximum stabilization. Would any of you have an ideal configuration to get the best look? 
Thank you in advance for your help:)

**Comment by dnewman-gpsw** [comment]
most video capture work fine in these conditions.  I'm sure there a skydiving forums elsewhere, this is a Labs firmware forum, so I expect it is unrelated to your question.
